pkg/plugins/common/k8s/names.go

```diff
@@ -15,8 +15,9 @@ const (
 	// The value has a format of a Kubernetes label name.
 	k8sNameComponent = "k8s.kuma.io/name"
 
-	// k8sSynced identifies that resource was synced
-	K8sSynced = "k8s.kuma.io/synced"
```

Comment:
Should we also remove this annotation from objects?

Reply:
Yes, I think we should, but I'd better do this in a separate PR, let me create an issue for that

Reply:
https://github.com/kumahq/kuma/issues/2542